design/design-doc-stacks.md

```diff
@@ -62,11 +63,15 @@ This section describes the end to end installation flow implemented by the Stack
     * package name (`gitlab`) OR
     * CRD name (`gitlabcluster.gitlab.com/v1alpha1`)
       * Note: this code path is exactly the same as dependency resolution
-* Performs dependency resolution that determines all packages/Stacks that are required by this Stack and all of its dependent Stacks
-* Pulls all necessary Stack packages from the registry
-* Unpacks them and installs all owned/defined CRDs and transforms the unpackaged content into an `Extension` CRD instance that serves as a record of the install
-* Starts up the custom controller so that it is in the running state
-* Marks the `ExtensionRequest` status as succeeded
+* The SM performs dependency resolution that determines all packages/Stacks that are required by this Stack and all of its dependent Stacks (Not Implemented)
+* The SM pulls all necessary Stack packages from the registry
+* The SM creates an unpack job and sends the artifacts to `stdout` which the SM ingests to install the Stack
+  * Stack metadata (`app.yaml`) is extracted and transformed to create an `Extension` CRD instance that serves as a record of the install
```

Comment:
technically, other metadata info is included into the `Extension` CRD instance.  The controller in `install.yaml` and the RBAC permissions in `rbac.yaml` are also included.